pkg/manager/member/tiflash_upgrader.go

```diff
@@ -101,6 +110,17 @@ func (u *tiflashUpgrader) Upgrade(tc *v1alpha1.TidbCluster, oldSet *apps.Statefu
 			if store.State != v1alpha1.TiKVStateUp {
 				return controller.RequeueErrorf("tidbcluster: [%s/%s]'s upgraded TiFlash pod: [%s], store state is not UP", ns, tcName, podName)
 			}
+			if ver, err := semver.NewVersion(tc.TiFlashVersion()); err == nil && ver.Compare(tiflashV512) >= 0 {
+				status, err := u.deps.TiFlashControl.GetTiFlashPodClient(tc.Namespace, tc.Name, podName, tc.IsTLSClusterEnabled()).GetStoreStatus()
```

Comment:
Please also cover the TLS enabled case during your test.

Reply:
ok

Reply:
done